op-service/sources/l1_beacon_client.go

```diff
@@ -140,7 +140,7 @@ func (cl *BeaconHTTPClient) BeaconBlobSideCars(ctx context.Context, fetchAllSide
 	}
 
 	if len(indices) > 0 {
-		return eth.APIGetBlobSidecarsResponse{}, fmt.Errorf("#returned blobs(%d) != #requested blobs(%d)", len(hashes)-len(indices), len(hashes))
+		return eth.APIGetBlobSidecarsResponse{}, fmt.Errorf("%s/%s?%s #returned blobs(%d) != #requested blobs(%d)", cl.cl.Endpoint(), reqPath, reqQuery.Encode(), len(hashes)-len(indices), len(hashes))
```

Comment:
Wondering if we rather want to log `resp.Request.RequestURI` or so, so we're debugging with the actual URL that was requested instead of reassembling something that may differ from what the http client actually requested?

From [docs](https://pkg.go.dev/net/http#Request)
```
 	// RequestURI is the unmodified request-target of the
	// Request-Line (RFC 7230, Section 3.1.1) as sent by the client
	// to a server. Usually the URL field should be used instead.
	// It is an error to set this field in an HTTP client request.
```